verify_imports.py

```
@@ -0,0 +1,87 @@
+
+        module = importlib.util.module_from_spec(spec)
+        sys.modules['temp_module'] = module
+
+        # This will execute imports but not the main code
+        # (assuming code is properly guarded with if __name__ == "__main__")
+        spec.loader.exec_module(module)
+
+        # Clean up
+        del sys.modules['temp_module']
+
+        return True, "OK"
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)